### PR TITLE
SSUT-227

### PR DESCRIPTION
**Pull request by versatil56**
- Repackaging all of consignors into its own package for `controllers,forms,views,pages` in both production code and also specs.
- Renaming `ConsignorsIdentity` to `ConsignorIdentity`

### CODE REVIEW

File: app/controllers/consignors/ConsignorIdentityController.scala [hunk 1]
```diff
@@ -14,31 +14,30 @@
...
-  view: ConsignorsIdentityView
+class ConsignorIdentityController @Inject()(
+                                             override val messagesApi: MessagesApi,
+                                             sessionRepository: SessionRepository,
+                                             identify: IdentifierAction,
+                                             getData: DataRetrievalActionProvider,
+                                             requireData: DataRequiredAction,
+                                             formProvider: ConsignorIdentityFormProvider,
+                                             val controllerComponents: MessagesControllerComponents,
+                                             view: ConsignorIdentityView
```
Comment (giftig) [inline app/controllers/consignors/ConsignorIdentityController.scala:40]: Can you run scalafmt please? You've put a big left gutter back in here.

File: app/controllers/consignors/ConsignorNameController.scala [hunk 2]
```diff
@@ -48,7 +47,7 @@ class ConsignorNameController @Inject() (
   def onPageLoad(mode: Mode, lrn: LocalReferenceNumber, index: Index): Action[AnyContent] =
     (identify andThen getData(lrn) andThen requireData) { implicit request =>
 
-      val preparedForm = request.userAnswers.get(ConsignorNamePage(index)) match {
+      val preparedForm = request.userAnswers.get(consignors.ConsignorNamePage(index)) match {
```
Comment (giftig) [inline app/controllers/consignors/ConsignorNameController.scala:50]: Why the change to no longer import the page itself?

File: app/viewmodels/checkAnswers/ConsignorIdentitySummary.scala [hunk 3]
```diff
@@ -16,35 +16,35 @@
 
 package viewmodels.checkAnswers
 
-import controllers.routes
+import controllers.consignors.{routes => consignorRoutes}
 import models.{CheckMode, Index, UserAnswers}
-import pages.ConsignorsIdentityPage
+import pages.consignors
```
Comment (giftig) [inline app/viewmodels/checkAnswers/ConsignorIdentitySummary.scala:21]: Here as well, imo we should just import the page, not a package.

File: app/viewmodels/checkAnswers/ConsignorIdentitySummary.scala [hunk 4]
```diff
@@ -16,35 +16,35 @@
 
 package viewmodels.checkAnswers
 
-import controllers.routes
+import controllers.consignors.{routes => consignorRoutes}
```
Comment (giftig) [inline app/viewmodels/checkAnswers/ConsignorIdentitySummary.scala:19]: I'd argue renaming route here is unnecessary; we're in a class specifically related to consignors already. (same everywhere else you've done the rename as well, I think it's all been in consignor-specific pages)

File: test-utils/generators/PageGenerators.scala [hunk 5]
```diff
@@ -18,7 +18,8 @@ package generators
 
 import models.Index
 import org.scalacheck.Arbitrary
-import pages._
+import pages.{consignors, _}
```
Comment (giftig) [inline test-utils/generators/PageGenerators.scala:21]: ?

File: test/pages/consignors/ConsignorAddressPageSpec.scala [hunk 6]
```diff
@@ -14,28 +14,29 @@
...
  */
 
-package pages
+package pages.consignors
 
 import base.SpecBase
 import controllers.routes
 import models.{Address, CheckMode, NormalMode}
 import pages.behaviours.PageBehaviours
+import pages.consignors
```
Comment (giftig) [inline test/pages/consignors/ConsignorAddressPageSpec.scala:23]: This is the package we're in already, why import this?

**Review by giftig — COMMENTED** [review]
Mostly just a few minor comments about importing

File: app/controllers/consignors/ConsignorIdentityController.scala (hunk 1 above)
Comment (versatil56) [reply app/controllers/consignors/ConsignorIdentityController.scala:40]: Yup! on it!

File: app/viewmodels/checkAnswers/ConsignorIdentitySummary.scala (hunk 4 above)
Comment (versatil56) [reply app/viewmodels/checkAnswers/ConsignorIdentitySummary.scala]: excellent spot, I had not moved this to its own package!

File: app/views/consignors/ConsignorAddressView.scala.html [hunk 7]
```diff
@@ -16,6 +16,7 @@
 
 @import models.Country._
 @import viewmodels.InputWidth._
+@import controllers.consignors.{routes => consignorRoutes}
```
Comment (giftig) [inline app/views/consignors/ConsignorAddressView.scala.html:19]: still a couple of renames in these few files which we could probably skip

**Review by giftig — APPROVED** [review]
